.appveyor.yml

```diff
@@ -5,6 +5,11 @@ environment:
 
 # Note that only Oracle JDK is provided.
   matrix:
+    - java: 9
```

Comment:
Main concerns about adding the Appveyor  matrix is: are we going to reach the limits of our usage and start to block PR checks once merged into`develop`? Do we have metrics about usage, wait & completion times?

Reply:
I have dropped two ant builds from both travis and appveyor so that it only builds with the primary supported jdk (8).